File: 1045.c
#include <stdio.h>
#include <math.h>
int main()
{
	double A,B,C,aux;
	scanf("%lf %lf %lf",&A,&B,&C);
	
	if(B>A && B>C){
		aux=A;
		A=B;
		B=aux;
	}
	if(C>A && C>B){
		aux=A;
		A=C;
		C=aux;
	}
	
		if(A>=B+C){
			printf("NAO FORMA TRIANGULO\n");
		}else{
		if(pow(A,2)==pow(B,2)+pow(C,2)){
			printf("TRIANGULO RETANGULO\n");
		}
		if(pow(A,2)>pow(B,2)+pow(C,2)){
			printf("TRIANGULO OBTUSANGULO\n");
		}
		if(pow(A,2)<pow(B,2)+pow(C,2)){
			printf("TRIANGULO ACUTANGULO\n");
		}
		if(A==B && B==C){
			printf("TRIANGULO EQUILATERO\n");
		}
		if(A==B && C!=A){
			printf("TRIANGULO ISOSCELES\n");
		}
		if(A==C && B!=A){
			printf("TRIANGULO ISOSCELES\n");
		}
		if(C==B && A!=B){
			printf("TRIANGULO ISOSCELES\n");
		}
		}
		
	return 0;
}

File: Em Python/1045.py
a, b, c = map(float, input().split())
if b > a and b > c:
     a = a + b
     b = a - b
     a = a - b
if c > a and c > b:
    a = a + c
    c = a - c
    a = a - c
if a >= b + c:
    print("NAO FORMA TRIANGULO")
else:
    if a * a == b * b + c * c:
        print("TRIANGULO RETANGULO")
    if a * a > b * b + c * c:
        print("TRIANGULO OBTUSANGULO")
    if a * a < b * b + c * c:
        print("TRIANGULO ACUTANGULO")
    if a == b == c:
        print("TRIANGULO EQUILATERO")
    if a == b and c != a:
        print("TRIANGULO ISOSCELES")
    if a == c and b != a:
        print("TRIANGULO ISOSCELES")
    if c == b and a != b:
        print("TRIANGULO ISOSCELES")